src/ClientInterface.php

```diff
@@ -21,5 +21,5 @@ interface Client
      * @param  array  $params Call parameters.
      * @return mixed Remote call results.
      */
-    public function call(string $method, array $params = []);
+    public function call($method, $params = []);
```

Comment:
Why are you removing the parameter typehints here?

Reply:
You are right, this makes no sense. I am not sure, what introduced this. Re-adding type declarations.